timezone.gemspec

```diff
@@ -25,8 +24,11 @@ Gem::Specification.new do |s|
   s.rdoc_options     = ['--charset=UTF-8']
   s.require_paths    = ['lib']
 
+  s.required_ruby_version = '>= 2.2'
+
   s.add_development_dependency('minitest', '~> 5.8')
   s.add_development_dependency('rake', '~> 12')
-  s.add_development_dependency('rubocop', '= 0.51')
+  s.add_development_dependency('rubocop', '>= 0.51')
```

Comment:
Can we clean up the warning logs related to RuboCop? I generally prefer to leave this pinned to one version to avoid having to upgrade syntax unnecessarily. I am fine if this gets upgraded from 0.51, but would prefer to leave it on a fixed version for development.

```
$ bundle exec rake
...
Running RuboCop...
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file....
```

Reply:
I've fixed the warnings, and locked the versions with a "<=".